each/Entities/EntityMuseum.py

```diff
@@ -80,7 +80,8 @@ def update_from_json(cls, data):
         eid = None
```

Comment:
Why did you make changes here? Do we try to add priority to the news, don't we? Please if you want to refactor other files - create another pull request :)

Reply:
This comment is for the whole file